poc/vidpf.py

```diff
@@ -212,20 +207,15 @@ def eval_next(cls, prev_seed, prev_ctrl, correction_word, cs_proof,
         for i in range(len(w)):
             y.append(w[i] + w_cw[i] * mask)
 
-        sha3 = hashlib.sha3_256()
         # pi' = H(x^{<= i} || s^i)
-        sha3.update(str(node).encode('ascii') +
-                    to_le_bytes(current_level, 2) + next_seed)
-        pi_prime = sha3.digest()
+        pi_prime = next_cs_proof(current_level, next_seed)
```

Comment:
Again, the node is getting removed here.

Reply:
Done.